api/valor_api/schemas/__init__.py

```diff
@@ -5,14 +5,17 @@
     EvaluationResponse,
 )
 from .filters import (
-    BooleanFilter,
-    DateTimeFilter,
+    Condition,
     Filter,
-    FilterQueryParams,
-    GeospatialFilter,
-    NumericFilter,
-    StringFilter,
-    convert_filter_query_params_to_filter_obj,
+    FilterOperator,
+    LogicalFunction,
+    LogicalOperator,
+    SupportedSymbol,
+    SupportedType,
+    Symbol,
+    Value,
+    soft_and,
```

Comment:
what are "soft_and" and "soft_or"? could these names be more descriptive?

Reply:
These functions handle the AND'ing of lists that were built using list comprehension and may not be valid.

I can rename them to `And` and `Or` since they basically are used as such.

Reply:
A better name could be `safe_and` and `safe_or`

Reply:
I called it `soft_and` b/c it would only create an `And` if there was at least 2 elements. Otherwise it would pass the object through without modifying it.

Reply:
IMHO, I'd be as specific as possible (e.g., `and_if_contains_two_elements`). `safe_and`/`safe_or` could also be "good enough" IMO

Reply:
I went in a different direction and implemented them as classmethods of `LogicalFunction`.

- `LogicalFunction.and_`
- `LogicalFunction.or_`
- `LogicalFunction.not_`